Server/VikingVault.Services/ScopedProcessingService.cs

```diff
@@ -46,6 +46,7 @@ public void PayCompanies()
                     automaticPayment.IsEnabled = false;
                     _dbContext.AutomaticPayments.Update(automaticPayment);
                     _dbContext.SaveChanges();
+                    AddFailedPaymentNotification(automaticPayment);
```

Comment:
AddFailedPaymentNotification if an update is succesful? It looks weird to me.

Reply:
How do you suggest to change that, considering that the update is made at first, so that the automatic payment will be disabled, and then we notify the user that the payment failed? Should I refactor the update in a separate function?